impeller/aiks/canvas.cc

```diff
@@ -725,6 +728,14 @@ void Canvas::SaveLayer(const Paint& paint,
   auto& new_layer_pass = GetCurrentPass();
   new_layer_pass.SetBoundsLimit(bounds);
 
+  if (paint.image_filter) {
+    MipCountVisitor mip_count_visitor;
+    paint.image_filter->Visit(mip_count_visitor);
+    new_layer_pass.SetRequiredMipCount(
+        std::max(mip_count_visitor.GetRequiredMipCount(),
```

Comment:
Maxing is not necessary here. If you place this logic in Save() then it will be easier to reason about.